### PR TITLE
dbr-14.3-support

### PR DESCRIPTION
**Pull request by panwroblewski**
- Add support for newer dbr version
- Make maxInflightBatches configurable or dependent on number of available cores

### CODE REVIEW

**Comment by panwroblewski** [comment]
@microsoft-github-policy-service agree

File: src/main/java/com/microsoft/pnp/client/GenericSendBuffer.java [hunk 1]
```diff
@@ -26,7 +27,9 @@ public abstract class GenericSendBuffer<T> implements AutoCloseable {
      */
     private final Semaphore inflightBatches;
     // Make configurable
-    private final int maxInflightBatches = 4;
+    private final int maxInflightBatches = Optional.ofNullable(System.getenv("SEND_BUFFER_MAX_INFLIGHT_BATCHES"))
+            .map(Integer::parseInt)
+            .orElseGet(() -> Runtime.getRuntime().availableProcessors());
```
Comment (FabianMeiswinkel) [inline src/main/java/com/microsoft/pnp/client/GenericSendBuffer.java:32]:
```suggestion
            .orElseGet(() -> Math.max(4, Runtime.getRuntime().availableProcessors()));
```

Comment (FabianMeiswinkel) [reply src/main/java/com/microsoft/pnp/client/GenericSendBuffer.java:32]: Just to be sure the default does not regress on any workers with < 4 Cores

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Comment by florent-brosse** [comment]
Thank you

**Pull request merged**